cfme/tests/configure/test_zones.py

```diff
@@ -121,3 +122,51 @@ def test_zone_add_blank_description():
             name=fauxfactory.gen_alphanumeric(5),
             description=''
         )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.sauce
+def test_add_zone_windows_domain_credentials(request):
+    """
+    Testing Windows Domain credentials add
+    """
+    zc = current_appliance.collections.zones.all()
+    values = {'username': 'userid',
+              'password': 'password',
+              'verify': 'password'}
+    zc[0].update(values)
```

Comment:
using default zone as this operation is safe and doesn't need any provider. cleanup is also here.